backend_new/src/main.ts

```diff
@@ -1,8 +1,17 @@
 import { NestFactory } from '@nestjs/core';
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  const config = new DocumentBuilder()
+    .setTitle('Bloom API')
+    .setDescription('The API for Bloom')
+    .setVersion('2.0')
+    .addTag('listings')
+    .build();
+  const document = SwaggerModule.createDocument(app, config);
+  SwaggerModule.setup('api', app, document);
   await app.listen(3000);
```

Comment:
I know this isn't part of your PR, but I think we should use a different port than 3000 so that it doesn't conflict with the public site. That doesn't have to be done in this PR, but at some point we should mimic backend/core where it takes the PORT from the env variable